apparmor.d/groups/kde/kwin_wayland_wrapper

```diff
@@ -18,10 +18,11 @@ profile kwin_wayland_wrapper @{exec_path} {
 
   @{bin}/kwin_wayland rPx,
 
-  owner @{run}/user/@{uid}/#@{int} rw,
-  owner @{run}/user/@{uid}/wayland-*.lock rk,
+  owner @{user_tmp_dirs}/#@{int} rw,
+  owner @{user_tmp_dirs}/xauth_@{rand6} w,
+  owner @{user_tmp_dirs}/wayland-*.lock rk,
```

Comment:
Don't use `@{user_tmp_dirs}` (it will most likely be removed in the future).

Reply:
Okay, I see I was following https://apparmor.pujol.io/development/guidelines/

Reply:
Yea, this is the exception... This variable was a test and the conclusion is that it gives access to too many things.